configs/components/cpp-pcp-client.rb

```diff
@@ -11,7 +11,12 @@
     pkg.environment "PATH", "#{settings[:bindir]}:/opt/pl-build-tools/bin:$(PATH)"
   end
 
-  pkg.build_requires "openssl"
+  if !settings[:vendor_openssl]
+    pkg.build_requires 'openssl-devel'
```

Comment:
You might need to do some switching based on OS, this isn't going to be the package name across all the supported OSes (or if it only supports EL*, we should fail if this is set for those platforms)